### ISSUE TITLE
The feature extraction technique (or the input to the neural network model) used in this implementation?

### ISSUE THREAD

**Issue by Xinghui-Wu** [alphacep/vosk-api#611]
The papers with regard to DeepSpeech 1 and 2 from Baidu claimed that they used the spectrogram of power normalized audio clips as the features to the system.
The implementation of Mozilla DeepSpeech, based on TensorFlow, said that they turned to apply the MFCC features.
Now I am curious about that of your implementation for my research purpose on the comparison of various feature extraction algorithms. So, I will appreciate it if you could solve my puzzle. Thank you.

**Comment by nshmyrev** [comment]
We use mfcc
